runtime/src/iree/hal/drivers/hip/hip_allocator.c

```diff
@@ -132,6 +183,7 @@ static iree_allocator_t iree_hal_hip_allocator_host_allocator(
 
 static iree_status_t iree_hal_hip_allocator_trim(
     iree_hal_allocator_t* IREE_RESTRICT base_allocator) {
+  // TODO implement
```

Comment:
```suggestion
  // TODO(awolosyzn): implement hip allocator trim.
```

include either your github ID or an issue number (`TODO(#1234)`) tracking the TODO, and include either a short bit about _what_ is to be done and optionally how (take notes for yourself and future people) - e.g. answer the question "what would trim do here?"

the idea is that we should be able to grep the codebase and roughly see the work to be done in various files - "implement" wouldn't be useful without context.

Reply:
Yep, this was a note to myself to implement this inside the WIP PR.